README.md

```diff
@@ -170,6 +180,27 @@ jobs:
 The action will automatically detect and use the Application Default
 Credentials.
 
+## With Connect gateway
```

Comment:
I couldn't find a spot for this but wanted to add some docs since this is a new feature. The heading above is for `## Authorization` of the action which I don't think this falls under. Other actions have some per action specific contents in the readme [as a separate heading](https://github.com/google-github-actions/deploy-cloudrun#allow-unauthenticated-requests).